velox/exec/tests/AggregationFuzzerTest.cpp

```diff
@@ -245,7 +245,8 @@ getCustomInputGenerators() {
 } // namespace facebook::velox::exec::test
 
 int main(int argc, char** argv) {
-  facebook::velox::aggregate::prestosql::registerAllAggregateFunctions();
+  facebook::velox::aggregate::prestosql::registerAllAggregateFunctions(
```

Comment:
Thanks. It would be nice to also cleanup skipFunctions and customVerificationFunctions to remove entries that correspond to companion functions.